create_pnor_image.pl

```diff
@@ -99,6 +104,10 @@
 print "scratch_dir = $scratch_dir\n";
 print "pnor_data_dir = $pnor_data_dir\n";
 
+if($xz_compression eq "false") {
+    run_command("sed -i '/<xz/d'  $xml_layout_file\n");
```

Comment:
This means we are modifying the original file, which is not ideal.  Can we easily put this into a tempfile instead?